tests/test-import.sh

```diff
@@ -123,6 +123,7 @@ CONTENT
 
 function printJsonValidationErrors {
   line=$1
+  [ -z "${line}" ] && return;
```

Comment:
Pour info: ce early exit permet d'éviter les erreurs en cas d'exécution de cette fonction sur une ligne vide. (`SyntaxError: Unexpected end of JSON input`)